src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -349,8 +350,19 @@ private ImmutableArray<Symbol> ComputeSortedCrefMembers(NamespaceOrTypeSymbol? c
                     diagnose: false,
                     useSiteInfo: ref useSiteInfo);
 
+                if (memberName is "nint" or "nuint"
```

Comment:
I'm just curious, why does this have to be specialized here and cannot go through the same code path that handles other type keywords like `int`? 🤔

Reply:
`nint` could bind to a type or alias. It's just an identifier, not a keyword.
We had to special case this in `BindNonGenericSimpleNamespaceOrTypeOrAliasSymbol` as well.

Reply:
Oh, I see, back compat 😔